.circleci/config.yml

```diff
@@ -3,29 +3,19 @@
 # Check https://circleci.com/docs/2.0/language-go/ for more details
 version: 2
 jobs:
-  markdown-lint-docs:
+  misspell-and-markdown-lint:
     docker:
-      # specify the version
-      - image: allencloud/mdl:v0.4
-    
-      # Specify service dependencies here if necessary
-      # CircleCI maintains a library of pre-built images
-      # documented at https://circleci.com/docs/2.0/circleci-images/
-      # - image: circleci/postgres:9.4
-
-    #### TEMPLATE_NOTE: go expects specific checkout path representing url
-    #### expecting it in the form of
-    ####   /go/src/github.com/circleci/go-tool
-    ####   /go/src/bitbucket.org/circleci/go-tool
+      - image: allencloud/mdlmisspell:v0.1
     working_directory: /go/src/github.com/{{ORG_NAME}}/{{REPO_NAME}}
     steps:
       - checkout
-
-      # specify any bash command here prefixed with `run: `
       - run:
           name: use markdownlint v0.4.0 to lint markdown file (https://github.com/markdownlint/markdownlint)
           command: |
             find  ./ -name  "*.md" | grep -v vendor | grep -v extra |  grep -v commandline |  grep -v .github |  grep -v swagger |  grep -v api |  xargs mdl -r ~MD010,~MD013,~MD024,~MD029,~MD033,~MD036
+      - run:
+          name: use opensource tool client9/misspell to correct commonly misspelled English words 
+          command: find  ./ -name  "*" | grep -v extra | grep -v vendor
```

Comment:
@allencloud Forgot to execute `misspell`? `xargs misspell -error`

Reply:
Ah, Thanks goodness. I am afraid I made a mistake in git commit.